src/detours.h

```diff
@@ -47,6 +47,7 @@
 #include <strsafe.h>
 #pragma warning(pop)
 #endif
+#include <crtdbg.h>
```

Comment:
Guard this include under #ifdef _DEBUG?

Reply:
No, crtdbg.h can check _DEBUG marco in its internal,
![image](https://user-images.githubusercontent.com/20564099/92315599-660a7a00-f01a-11ea-91ba-62377e338fe2.png)
this macro and function is ported from my personal project many year ago,  so it has be tested many times and many years.

![image](https://user-images.githubusercontent.com/20564099/92316099-0617d180-f022-11ea-8da7-85a80a7bcd37.png)

![image](https://user-images.githubusercontent.com/20564099/92316092-ebddf380-f021-11ea-90d8-036e253ca762.png)

`_CrtDbgReport` is basic of CRT's _ASSERTE, ATL's ATLASSERT, and MFC's ASSERT, but _CrtDbgReportW can only be avaliable in high version visual c, so I decide to use _CrtDbgReport at that time.